book/src/drivers.md

```diff
@@ -41,7 +41,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Hetzner Cloud Volumes CSI](https://github.com/hetznercloud/csi-driver) | `csi.hetzner.cloud` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Hetzner Cloud Volumes | Persistent | Read/Write Single Pod | Yes | Expansion
 [Hitachi Vantara](https://knowledge.hitachivantara.com/Documents/Adapters_and_Drivers/Storage_Adapters_and_Drivers/Containers) | `com.hitachi.hspc.csi` | v1.0 | A Container Storage Interface (CSI) Driver for VSP series Storage | Persistent | ? | ? | 
 [HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent and Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
-[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Huawei Storage](https://support.huawei.com/enterprise/en/index.html) | Persistent | Read/Write Multiple Pod | Yes | 
+[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [FusionStorage, OceanStor 100D, ](https://support.huawei.com/enterprise/en/category/cloud-storage-pid-1482615290338?submodel=21430817)[OceanStor Dorado V3, OceanStor Dorado V6, OceanStor V3, OceanStor V5](https://support.huawei.com/enterprise/en/category/enterprise-storage-pid-1482606575525?submodel=21430818) | Persistent | Read/Write Multiple Pod | Yes | 
```

Comment:
The display does not look right.  Can you take a look of "IBM Block Storage" as an example of showing multiple storage types?

Reply:
See the display here.  It's messed up: https://github.com/kubernetes-csi/docs/blob/28bb27135a64d8b1d5d7acf8f995ca792f40bf88/book/src/drivers.md